src/components/AccountDrawer/HideSpamToggle.tsx

```diff
@@ -0,0 +1,19 @@
+import { t } from '@lingui/macro'
+import { useAtom } from 'jotai'
+import { atomWithStorage } from 'jotai/utils'
+
+import { SettingsToggle } from './SettingsToggle'
+
+export const hideSpamAtom = atomWithStorage<boolean>('hideSmallBalances', true)
+
+export function HideSpamToggle() {
+  const [hideSpam, updateHideSpam] = useAtom(hideSpamAtom)
+
+  return (
+    <SettingsToggle
+      title={t`Hide unknown tokens & NFTs`}
```

Comment:
I feel like i've been peripherally aware of conversation that we prefer <Trans> over t`?